holopy/scattering/theory/dda.py

```diff
@@ -73,11 +73,13 @@ class DDA(ScatteringTheory):
     excessive memory or computation time for particularly large scatterers.
     """
     def __init__(self, n_cpu = 1, max_dpl_size=None, use_indicators=True,
-                 keep_raw_calculations=False, addacmd=[]):
+                 keep_raw_calculations=False, addacmd=[],
+                 suppress_C_output=True):
 
         # Check that adda is present and able to run
         try:
-            subprocess.check_call(['adda', '-V'])
+            with SuppressOutput():
```

Comment:
Thanks for suppressing output on the DDA tests! I meant to do this when I suppressed the output on the tmatrix tests but I forgot to do so.